### PR TITLE
connects Falkonry/tracker#1053

### PR DESCRIPTION
**Pull request by dilpeshjain**

### CODE REVIEW

File: test/FalkonryTests.cs [hunk 1]
```diff
@@ -3117,6 +3119,70 @@ public void TestHistoricalOutput()
         }
     }
 
+     [TestClass]
+    public class GetStreamingOutput
+    {
+        internal class FalkonryEvent
+        {
+            // ReSharper disable once InconsistentNaming
+            public string entity { get; set; }
```
Comment (phagunbaya) [inline test/FalkonryTests.cs]: Need a field for `batch` too

File: test/FalkonryTests.cs [hunk 2]
```diff
@@ -3117,6 +3119,70 @@ public void TestHistoricalOutput()
...
+            public string time { get; set; }
+            // ReSharper disable once InconsistentNaming
+            public string value { get; set; }
+            public override string ToString()
+            {
+                return $"{{time: '{time}', entity: '{entity}', value: '{value}'}}";
+            }
+        }
+        //Falkonry _falkonry = new Falkonry("https://localhost:8080", "auth-token");
+        Falkonry _falkonry = new Falkonry("https://dev.falkonry.ai", "n4qlyqyl7eejz9i2sc1bpi5bz6ry3wvx");
```
Comment (phagunbaya) [inline test/FalkonryTests.cs]: DO NOT commit this.

File: test/FalkonryTests.cs [hunk 3]
```diff
@@ -3117,6 +3119,70 @@ public void TestHistoricalOutput()
...
+                Assert.AreNotEqual(falkonryEvent.value, null);
+            }
+            catch(System.Exception exception)
+            {
+                Assert.AreEqual(exception.Message, null, false);
+            }
+            
+        }
+
+        private void EventSource_NewError(object sender, EventSource.ServerSentErrorEventArgs e)
```
Comment (phagunbaya) [inline test/FalkonryTests.cs]: rename it to `error`

File: test/FalkonryTests.cs [hunk 4]
```diff
@@ -3117,6 +3119,70 @@ public void TestHistoricalOutput()
...
+            Dictionary<string, EventSource> _eventSource = new Dictionary<string, EventSource>();
+            try
+            {
+
+                EventSource eventSource = null;
+                eventSource = _falkonry.GetOutput(assessment,null,null);
+                eventSource.Message += EventSource_Message;
+                eventSource.Error += EventSource_NewError;
+                _eventSource.Add(assessment, eventSource);
+                Thread.Sleep(60000);
```
Comment (phagunbaya) [inline test/FalkonryTests.cs]: Why do you need sleep here ?

Comment (dilpeshjain) [reply test/FalkonryTests.cs]: We don't get output immediately, we need to wait for sometime to receive the output.

File: README.md [hunk 5]
```diff
@@ -1025,23 +1025,7 @@ Sample CSVFile
...
-    System.IO.StreamWriter streamwriter = new System.IO.StreamWriter(path);
-    string line;
-    using (streamwriter)
-    {
-        while ((line = streamreader.ReadLine()) != null)
-        {
-            streamwriter.WriteLine(line);
-        }
-    }
+    EventSource eventSource = falkonry.getOutput(assessment_id, null, null);
```
Comment (phagunbaya) [inline README.md]: Need to show how to add bind functions for message and error. Better to write a test case instead of a small snippet.

**Pull request merged**